tests/OSPSuite.Presentation.Tests/Importer/Presenters/ImporterPresenterSpecs.cs

```diff
@@ -415,4 +418,32 @@ public void must_validate_mapping()
          A.CallTo(() => _columnMappingPresenter.ValidateMapping()).MustHaveHappened();
       }
    }
+
+   public class When_loading_configuration : concern_for_ImporterPresenter
+   {
+
+      protected override void Because()
+      {
+         var sheets = new Cache<string, DataSheet>();
```

Comment:
This is more CONTEXT of the test. .I am having trouble seeing what causes the configuration load. Can you separate Context from action?

Reply:
I hope it makes more sense now?